tests/tests.rs

```diff
@@ -290,6 +314,7 @@ fn create_bindgen_builder(header: &PathBuf) -> Result<Builder, Error> {
 
     // Scoop up bindgen-flags from test header
     let mut flags = Vec::with_capacity(2);
+    let mut parse_callbacks = vec![];
```

Comment:
Hmm, is there any need for handling multiple of these? The last one will just win right? Can this just use an `Option`?